test_data/saml_config.yml

```diff
@@ -0,0 +1,14 @@
+---
+IDP_METADATA_PATH: |
+    <xml> this is some idp xml data </xml>
+DISPLAY_NAME_ATTRIBUTE: 'def'
+GROUP_ATTRIBUTE: 'ghi'
+MAX_AUTH_LIFETIME: 10
```

Comment:
What unit is this measured in?  Days?  Can that be added to the variable name for clarity?